<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModelInferUI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Model Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_modeltype"/>
       </item>
       <item>
        <widget class="QPushButton" name="button_imgdir">
         <property name="text">
          <string>Image Dir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_imgdir"/>
       </item>
       <item>
        <widget class="QPushButton" name="button_modeldir">
         <property name="text">
          <string>Model Dir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_modeldir"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,1">
         <item>
          <widget class="QGroupBox" name="groupBox_setting">
           <property name="title">
            <string>Infer Setting</string>
           </property>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>91</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Model Num</string>
            </property>
           </widget>
           <widget class="QComboBox" name="combo_modelnum">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>30</y>
              <width>111</width>
              <height>25</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Gpu Id</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>71</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Cycles</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_gpuid">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>60</y>
              <width>113</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_cycles">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>90</y>
              <width>113</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>120</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>New W</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_newW">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>120</y>
              <width>113</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>512</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>120</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>New H</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_newH">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>120</y>
              <width>113</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>512</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>160</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Warmup</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>160</y>
              <width>91</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Save Result</string>
            </property>
           </widget>
           <widget class="Line" name="line">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>190</y>
              <width>351</width>
              <height>16</height>
             </rect>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>210</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Use trt</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="check_warmup">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>160</y>
              <width>92</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="check_saveresult">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>160</y>
              <width>92</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QCheckBox" name="check_usetrt">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>210</y>
              <width>92</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>240</y>
              <width>121</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Min Shape(W*H)</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_11">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>270</y>
              <width>111</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Opt Shape(W*H)</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_12">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>300</y>
              <width>121</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Max Shape(W*H)</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_13">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>360</y>
              <width>67</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>precision</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_minshapeW">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>240</y>
              <width>113</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_minshapeH">
            <property name="geometry">
             <rect>
              <x>262</x>
              <y>240</y>
              <width>111</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_optshapeW">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>270</y>
              <width>113</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_optshapeH">
            <property name="geometry">
             <rect>
              <x>262</x>
              <y>270</y>
              <width>111</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_maxshapeW">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>300</y>
              <width>113</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_maxshapeH">
            <property name="geometry">
             <rect>
              <x>262</x>
              <y>300</y>
              <width>111</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
           <widget class="QComboBox" name="combo_precision">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>360</y>
              <width>111</width>
              <height>25</height>
             </rect>
            </property>
           </widget>
           <widget class="Line" name="line_2">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>380</y>
              <width>351</width>
              <height>16</height>
             </rect>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QPushButton" name="button_updatemodel">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>400</y>
              <width>111</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>Update Model</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_14">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>330</y>
              <width>131</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Min Subgraph Size</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_minsubgraph">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>330</y>
              <width>113</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
           <item>
            <widget class="QPushButton" name="button_start">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_visualize">
             <property name="text">
              <string>Visualization</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_log"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
